curacmd/install

```diff
@@ -1 +1,150 @@
-# Curacmd Bootstrap / Install Script
+#!/bin/bash
```

Comment:
If this is your first Bash script then bravo! And those kids don't know what they're talking about, extensions are where it's at.